svelte/package.json

```diff
@@ -0,0 +1,69 @@
+{
+	"name": "uplot-svelte",
+	"version": "1.1.5",
+	"description": "Svelte wrapper for uPlot that allows you to work with charts declaratively inside your favorite framework",
+	"author": "Kiril Panayotov <contact@zakrok.dev> (https://zakrok.dev)",
+	"contributors": [
+		{
+			"name": "Sergey Kalinichev",
+			"email": "kalinichev.so.0@gmail.com"
+		}
+	],
+	"license": "MIT",
+	"homepage": "https://github.com/skalinichev/uplot-wrappers#readme",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/skalinichev/uplot-wrappers.git"
+	},
+	"bugs": {
+		"url": "https://github.com/skalinichev/uplot-wrappers/issues"
+	},
+	"keywords": [
+		"Charts",
+		"uPlot",
+		"Svelte",
+		"SvelteKit",
+		"Wrapper"
+	],
+	"scripts": {
+		"dev": "vite dev",
```

Comment:
Can we use webpack for that? The idea is to use the same tools for all wrappers. While it may be not that convenient, it keeps the code-base consistent. It would be really difficult for me to maintain it if every wrapper uses it's own tools(
Please, see if you can do it, otherwise i'll have to do it myself later on.

Reply:
I attempted to do it myself, I believe it is very much possible, the only issue is that I am having trouble with the existing webpack.configuration. I leave it as vite for now because I am afraid I cannot do it myself.

I will instead link articles and packages that I believe would be very helpful:

- [Svelte + Webpack 5 article](https://codechips.me/svelte-and-webpack-5/)
- [svelte-loader](https://github.com/sveltejs/svelte-loader)
- [svelte-preprocess](https://github.com/sveltejs/svelte-preprocess)